GliaWidgets/Sources/EntryWidget/EntryWidgetViewModel.swift

```diff
@@ -0,0 +1,26 @@
+import SwiftUI
+
+extension EntryWidgetView {
+    class Model: ObservableObject {
```

Comment:
Just to clarify, this is observable object, but it does not have any published properties yet, is this correct?

Reply:
As of yet, there no published properties in the class. This is due to the business logic being in development still. There most likely will be some, and that's why @observableObject was added proactively.